spec/unit/tracker_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'spec_helper'
 
 describe Mongoid::History::Tracker do
+  before { Mongoid::History.tracker_class_name = nil }
```

Comment:
If you're going to set something global in a spec, reset it to the old value in `after`.